.github/dependabot.yml

```
@@ -7,3 +7,8 @@ updates:
     # Check the npm registry for updates every day (weekdays)
     schedule:
       interval: 'daily'
+
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```

```suggestion
      interval: 'weekly’
```

I recommend checking weekly for the reason described in https://github.com/actions/stale/pull/756.